index.json

```diff
@@ -1,17 +1,6 @@
 {
-  "printWidth": 80,
-  "tabWidth": 2,
-  "useTabs": false,
+  "useTabs": true,
```

Comment:
Tabs take up less space (in terms of bytes), most editors allow for configuration of custom width per tab, making them more accessible.

Devs can still write code with spaces and it'll get transformed by the precommit hooks.

Reply:
Is the default `tabWidth` now `2`?

Reply:
Yep! https://prettier.io/docs/en/options.html#tab-width